config.yaml

```diff
@@ -2,4 +2,8 @@ options:
   credentials:
     type: string
     default: ""
-    description: "Space separated username and password for basic authentication."
\ No newline at end of file
+    description: "Space separated username and password for basic authentication."
+  dhparam:
+    type: string
+    default: "/etc/nginx/dhparam/dhparam.pem"
```

Comment:
Ok, I understand. How can we solve this without needing operators to do this manually? Can we just generate different params for each site and prefix the file with the site name with something unique?